saga/register-credit-connector.json

```diff
@@ -0,0 +1,19 @@
+{
+    "connector.class": "io.debezium.connector.postgresql.PostgresConnector",
+    "tasks.max": "1",
+    "database.hostname": "customer-db",
+    "database.port": "5432",
+    "database.user": "customeruser",
+    "database.password": "customerpw",
+    "database.dbname" : "customerdb",
+    "database.server.name": "dbserver3",
+    "schema.include.list": "customer",
+    "table.include.list" : "customer.outboxevent",
+    "tombstones.on.delete" : "false",
+    "key.converter": "org.apache.kafka.connect.storage.StringConverter",
+    "value.converter": "org.apache.kafka.connect.storage.StringConverter",
+    "value.converter.schemas.enable": "false",
```

Comment:
since value is plain String this is not needed I guess?

Reply:
Indeed, that's a left-over from earlier tries.